tests/playwright-tests/admin/azpage/AzLoginPage.ts

```diff
@@ -0,0 +1,35 @@
+import { Locator, Page } from '@playwright/test';
+import  environment  from '../../utils/env';
+
+// AzureLogin Page
+export default class AzureLoginPage {
```

Comment:
Given this class name, would suggest we name the file `AzureLoginPage.ts` to reflect this as well